tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -477,4 +463,23 @@ public function testEnsureDefaultsAreUnescapedFromSchemaIntrospection() : void
         $onlineTable = $this->_sm->listTableDetails("test_column_defaults_with_create");
         self::assertSame($default, $onlineTable->getColumn('col1')->getDefault());
     }
+
+    protected function getEscapedLiterals() : array
```

Comment:
There should be one test for testing a unified set of special characters on all supported platforms. Testing different sets on different platforms doesn't guarantee that the API is portable.

Reply:
Fixed.